chunky/src/java/se/llbit/chunky/chunk/BlockPalette.java

```diff
@@ -135,6 +135,12 @@ public Block get(int id) {
     return palette.get(id);
   }
 
+  public Tag getTag(int id) {
+    return blockMap.entrySet().stream().filter(item -> item.getValue() == id)
+        .map(item -> item.getKey().getTag())
+        .findFirst().orElse(null);
```

Comment:
Replace with a `foreach` loop? This stream is very hard to comprehend.

Reply:
Is it? It's just a simple filter/map, maybe I'm just more used to this stuff… I find it way easier to read than a for loop

Reply:
I find it challenging to understand, I think a simple `foreach` and `if` would be very obvious as to what it does.

Reply:
Okay, changed it :+1: